XCTestPlans/AWSIntegrationTestsOnCI.xctestplan

```diff
@@ -0,0 +1,95 @@
+{
+  "configurations" : [
+    {
+      "id" : "FF4C1A8F-5D70-4A2A-BFDE-77467500C547",
+      "name" : "Test Scheme Action",
+      "options" : {
+      }
+    }
+  ],
+  "defaultOptions" : {
+    "environmentVariableEntries" : [
+
+    ]
```

Comment:
The step defined in the Github workflow above will fill this empty JSON array with environment variable settings for the test run's AWS credentials.